zppy/templates/coupled_global.py

```diff
@@ -399,6 +472,7 @@ def plot(ax, xlim, exps, param_dict, rgn):
         extreme_values.append(np.amin(var))
         if param_dict["shorten_year"]:
             year = year[: len(var)]
+        print(f"extreme_values={extreme_values}")
```

Comment:
Note to self 2) review irrelevant print statements. (Some print statements may be relevant/useful to users debugging output)

Reply:
Fixed.